### PR TITLE
Create resource request documentation

### PR DESCRIPTION
**Pull request by annabellegoth2boss**
Initial draft: Documents how OpenSSF projects can request the resources/benefits outlined in the Project Lifecycle document. cc @AevaOnline for review and input.

- [ ] @strawberry-baked-alaska Please review section on when to engage Marketing Committee
- [ ] @david-a-wheeler @brianbehlendorf Please see section on infrastructure. Many ways this could be structured, but I imagine it's similar to the CNCF Cloud Credits program wants to do (cc @caniszczyk)

Re: Budget Committee requests, projects and members will need to be able to see a calendar of meetings, members and agenda etc to submit requests

### CODE REVIEW

**Review by SecurityCRob — COMMENTED** [review]
No suggest d changes
+1

File: process/request_resources.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+### Swag
+
+The OpenSSF Marketing Committee keeps a small, annual swag stipend for all projects. Projects may wish to seek outside funding for project swag beyond this stipend.
+
+## Requests through the OpenSSF Budget Committee
+Requests that go through the OpenSSF Budget Committee are reviewed at regularly occuring Budget Committee meetings. Projects should plan their funding needs and requests with this timeline in mind.
+
+### Infrastructure
+
+_Program TBD as of June 2022. Intended program details below._
```
Comment (inferno-chromium) [inline process/request_resources.md:33]: These two _Program TBD can be removed?

File: process/request_resources.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+All OpenSSF projects receive support and resources from the Foundation. The type of resources and amount vary by project stage; refer to the [project lifecycles](process/project_lifecycle.md) for details. 
+
+- Common requests
+    - [Blogs](#communications-and-content)
+    - [Event space](#contribuor-meet-ups-at-OpenSSF-events)
+    - [Infrastructure](#infrastructure)
+    - [Security audits](#security-audits-and-improvements)
+    - [Code of Conduct support](#code-of-conduct-support)
+
+## Requests through the OpenSSF Marketing WG
```
Comment (inferno-chromium) [inline process/request_resources.md:12]: I don't think there is a Marketing WG, probably just Marketing Committee (which is just Jenn atm)?

File: process/request_resources.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+### Maintainer stipends
+
+_Program TBD as of June 2022. Intended program details below._
+Limited, time-bound maintainer stipends for specified project improvements may be available to qualifying projects. Projects should submit a request to the OpenSSF Budget Committee with proposed scope, cost, desired outcome, TAC sponsor approval, and timeline at least three months before the requested start date. Maintainer stipends may not be paid to employees of OpenSSF member companies (excludes Associate members).
+
+### Security audits and improvements
+
+Qualifying projects may request funding for security audits and other services offered by third party security professionals. Before creating a request for proposals, projects should confirm funding availablity with the OpenSSF Budget Committee. After receiving proposals, projects should request their selected proposal be funded through the Budget Committee. Projects who need assistance creating an RFP should contact the OpenSSF GB Planning Committee. RFPs must be posted in a public channel(s) (GitHub issue, mailing list). OpenSSF member companies may bid on RFPs.
```
Comment (inferno-chromium) [inline process/request_resources.md:43]: s/OpenSSF GB Planning Committee/OpenSSF Planning Committee (without the GB?)

File: process/request_resources.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+All OpenSSF projects receive support and resources from the Foundation. The type of resources and amount vary by project stage; refer to the [project lifecycles](process/project_lifecycle.md) for details. 
+
+- Common requests
+    - [Blogs](#communications-and-content)
+    - [Event space](#contribuor-meet-ups-at-OpenSSF-events)
+    - [Infrastructure](#infrastructure)
+    - [Security audits](#security-audits-and-improvements)
+    - [Code of Conduct support](#code-of-conduct-support)
+
```
Comment (inferno-chromium) [inline process/request_resources.md:11]: Should the requests through Marketing and Budget committee first get review and approval from TAC ?

**Review by inferno-chromium — APPROVED** [review]
LGTM with some comments/questions

File: process/request_resources.md (hunk 2 above)
Comment (AevaOnline) [reply process/request_resources.md:12]:
```suggestion
## Requests through the OpenSSF Marketing Committee
```

File: process/request_resources.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+## Requests through the OpenSSF Marketing WG
+
+### Communications and content
+
+To post on the OpenSSF blog, projects should contact the OpenSSF Marketing WG with a blog draft two weeks before the requested publication date. The WG does their best to accomodate requested publication dates, but projects should plan to be flexible during high-traffic times (e.g. events). Posts and timing is at the editorial discretion of the Marketing WG; posts must follow the Code of Conduct, be relevant to the OpenSSF mission, and not promote vendor or commercial products. 
+
+To receive space in an OpenSSF newsletter, projects should contact the Marketing WG a minimum of two weeks before the newsletter publication date. Projects are responsible for tracking due dates and submitting materials.
```
Comment (AevaOnline) [inline process/request_resources.md:18]:
```suggestion
To receive space in an OpenSSF newsletter, projects should contact the Marketing Committee a minimum of two weeks before the newsletter publication date. Projects are responsible for tracking due dates and submitting materials.
```

File: process/request_resources.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+    - [Code of Conduct support](#code-of-conduct-support)
+
+## Requests through the OpenSSF Marketing WG
+
+### Communications and content
+
+To post on the OpenSSF blog, projects should contact the OpenSSF Marketing WG with a blog draft two weeks before the requested publication date. The WG does their best to accomodate requested publication dates, but projects should plan to be flexible during high-traffic times (e.g. events). Posts and timing is at the editorial discretion of the Marketing WG; posts must follow the Code of Conduct, be relevant to the OpenSSF mission, and not promote vendor or commercial products. 
+
+To receive space in an OpenSSF newsletter, projects should contact the Marketing WG a minimum of two weeks before the newsletter publication date. Projects are responsible for tracking due dates and submitting materials. 
+
```
Comment (AevaOnline) [inline process/request_resources.md:19]:
```suggestion
Technical content should be reviewed by the TAC, and potentially other WG's or Committees as appropriate. This includes, but is not limited to: white papers, technical briefings, policy recommendations, etc. This does not include project-specific announcements, such as those relating to project milestone releases. 

```

File: process/request_resources.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+### Security audits and improvements
+
+Qualifying projects may request funding for security audits and other services offered by third party security professionals. Before creating a request for proposals, projects should confirm funding availablity with the OpenSSF Budget Committee. After receiving proposals, projects should request their selected proposal be funded through the Budget Committee. Projects who need assistance creating an RFP should contact the OpenSSF GB Planning Committee. RFPs must be posted in a public channel(s) (GitHub issue, mailing list). OpenSSF member companies may bid on RFPs. 
+
+## Other
+
+### Code of Conduct support
+
+All projects must follow the [OpenSSF Code of Conduct](https://openssf.org/community/code-of-conduct/). If projects receive reports of conduct violations, they should report them to conduct@openssf.org.
```
Comment (AevaOnline) [inline process/request_resources.md:49]: Some projects, especially those that already have a community, may have a different CoC, and we should allow that if and only if they have a working CoC body to back it up.

I'm unsure the best organizational check-and-balance to add a this time, so I propose putting this on the TAC, as it is a project governance question, not a financial or legal question.

```suggestion
All projects must follow the [OpenSSF Code of Conduct](https://openssf.org/community/code-of-conduct/), or, if a project has its own Code of Conduct, it must be reviewed and approved by the TAC. 

If projects receive reports of conduct violations, they should report them to conduct@openssf.org.
```

File: process/request_resources.md (hunk 6 above)
Comment (annabellegoth2boss) [reply process/request_resources.md:19]: +1

File: process/request_resources.md (hunk 7 above)
Comment (annabellegoth2boss) [reply process/request_resources.md:50]: +1 for now. Want to make sure the TAC is equipped to do this (or if it's another body like the Planning Committee), but +1 to having a process that acknowledges existing project CoCs.

**Review by AevaOnline — APPROVED** [review]

**Pull request merged**